tools/helpers.py

```diff
@@ -0,0 +1,191 @@
+import os
+import logging
+from configparser import ConfigParser
+import subprocess
+
+
+def setup_logger(name, log_path=None):
+    """Enables the logging to be setup correctly"""
+    logger = logging.getLogger(name)
+    logger.setLevel(logging.DEBUG)
+
+    stream_handle = logging.StreamHandler()
+    stream_handle.setLevel(logging.DEBUG)
+    stream_handle.setFormatter(
+        logging.Formatter("%(asctime)s - %(levelname)s - %(message)s")
+    )
+    logger.addHandler(stream_handle)
+
+    if log_path:
+        file_handle = logging.FileHandler(log_path, "a")
+        file_handle.setLevel(logging.DEBUG)
+        file_handle.setFormatter(
+            logging.Formatter("%(asctime)s - %(levelname)s - %(message)s")
+        )
+        logger.addHandler(file_handle)
+
+    return logger
+
+
+def get_config():
+    """
+    Read in a config.ini file and return the object. Assumes it's located in
+    the wrapper root folder and called config.ini
+    """
+    converters = {"list": lambda x: [i.strip() for i in x.split(",")]}
+    config = ConfigParser(converters=converters)
+
+    root_path = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+    config_path = os.path.join(root_path, "config.ini")
+
+    if not os.path.exists(config_path):
+        raise FileNotFoundError(f"No config found at {config_path}")
+
+    config.read(config_path)
+    return config
+
+
+def dir_to_samplesheet(scriptpath: str, fastqdir: str, strandedness: str) -> str:
+    """
+    Executes the nf-core fastq_dir_to_samplesheet.py script on a given dir with fastq files
+    :param scriptpath: Path to fastq_dir_to_samplesheet.py
+    :param fastqdir: Path to directory with fastq files
+    :param strandedness: Strandedness of library
+    :return: Path to the generated samplesheet.csv
+    """
+    ss_path = os.path.join(fastqdir, "samplesheet.csv")
```

Comment:
samplesheet name should be case insensitive and able to tolerate other permutations like SampleSheet_blahextrainfo123.csv and similar

Reply:
In this case samplesheet is always generated by this repo, so it's not a samplesheet from the lab. The name will always be samplesheet.csv. But I will make it case insensitive just in case!

Reply:
Fixed in ee01cdc